code/game/machinery/doors/poddoor.dm

```diff
@@ -74,12 +87,10 @@
 /obj/machinery/door/poddoor/ert
 	name = "hardened blast door"
 	desc = "A heavy duty blast door that only opens for dire emergencies."
-	ertblast = TRUE
```

Comment:
I don't see anything replicating this functionality.

Reply:
i think that functionality was made obsolete when they made it so you can only open maintenance panel while the blast door is open. so you can't deconstruct ERT blast doors before they get opened